heimdall2/values.yaml

```diff
@@ -168,15 +174,84 @@ certs:
   command: "['sh', '-c', 'update-ca-trust']" # RHEL style distros use the `update-ca-trust` command to update the system CA trust store
   injectedCertsMountPath: /etc/pki/ca-trust/source/anchors # the location that `update-ca-trust` looks inside for user supplied certs
   processedCertsMountPaths: [/etc/pki/ca-trust/extracted/openssl, /etc/pki/ca-trust/extracted/pem, /etc/pki/ca-trust/extracted/java, /etc/pki/ca-trust/extracted/edk2] # due to the chart using `emptyDir` (which supplies a completely blank dir) to specify the shared volumes between this container and Heimdall's, and due to `update-ca-trust` apparently requiring several subdirectories underneath /etc/pki/ca-trust/extracted to exist, we need to manually specify each subdirectory
-  # securityContext: {
-  # RunAsRoot: false
-  # }
-  # resources: {
-  # }
+#   securityContext: {}
+#   runAsRoot: false
+#   resources: 
+#   
   certificates:
```

Comment:
we should provide an explanation here clarifying the difference between the configmap and initcontainer approach